Dockerfile.bundle

```diff
@@ -8,9 +8,6 @@ RUN dnf install -y python3 python3-ruamel-yaml-clib python3-ruamel-yaml patch
 RUN mkdir /licenses
 COPY tempo-operator/LICENSE /licenses/.
 
-# Use a new stage to enable caching of the package installations for local development
-FROM builder-deps as builder
```

Comment:
why this change?

Reply:
Container layers are already cached unless a modified context is copied to a layer with `COPY`, but in this case the `COPY` is after the package installation, i.e. the installation layers are always cached.

Do you know why it was there in the first place?